Quick/QuickSpec.m

```diff
@@ -101,7 +101,11 @@ + (SEL)addInstanceMethodForExample:(Example *)example {
         [example run];
     });
     const char *types = [[NSString stringWithFormat:@"%s%s%s", @encode(id), @encode(id), @encode(SEL)] UTF8String];
-    SEL selector = NSSelectorFromString(example.name.qck_selectorName);
+    NSString *selectorName = [NSString stringWithFormat:@"%@_%@_%ld",
+                              example.name.qck_selectorName,
+                              example.callsite.file,
+                              (long)example.callsite.line];
+    SEL selector = NSSelectorFromString(selectorName);
```

Comment:
My question is: Isn't the new selector name much to long? Shouldn't we e.g. strip the path and only use the filename?